src/Helper/IrisHelper.php

```diff
@@ -3,6 +3,7 @@
 namespace ElevenMiles\Iris\Helper;
 
 define('IRIS_ASSET_PATH', get_site_url() . '/vendor/eleven-miles/iris/src/assets');
```

Comment:
Where does this const get used? Can see it referenced in `get_asset_path` but not sure where that method is used? Only ask as ideally would try to avoid the `get_site_url()` method here in the path as it will potentially be impacted by domain changes or to the web root.

If this is required for Iris specific assets i.e. within the admin, then perhaps we can look at alternative methods to loading associated files?